sdk/schemaregistry/azure-schemaregistry-avroencoder/samples/README.md

```diff
@@ -0,0 +1,65 @@
+---
+page_type: sample
+languages:
+  - python
+products:
+  - azure
+  - azure-event-hubs
+urlFragment: schemaregistry-avroencoder-samples
+---
+
+# Azure Schema Registry Avro Encoder library for Python Samples
+
+These are code samples that show common scenario operations with the Schema Registry Avro Encoder library.
+
+Several Schema Registry Avro Encoder Python SDK samples are available to you in the SDK's GitHub repository. These samples provide example code for additional scenarios commonly encountered while working with Schema Registry Avro Encoder:
+
+* [encode_and_decode_event_data_message.py][encode_and_decode_event_data_message_sample] ([async version][encode_and_decode_event_data_message_async_sample]) - Examples for common Schema Registry Avro Encoder tasks:
+    * Encode data according to the given schema and create EventData object
+    * Decode data given an EventData object with encoded data and corresponding content type
+* [eventhub_send_integration.py][eventhub_send_integration_sample] ([async version][eventhub_send_integration_async_sample]) - Examples for integration with EventHub in sending tasks:
+    * Encode data with the given schema and send `EventData` to Event Hubs.
+* [eventhub_receive_integration.py][eventhub_receive_integration_sample] ([async version][eventhub_receive_integration_async_sample]) - Examples for integration with EventHub in receiving tasks:
+    * Receive `EventData` from Event Hubs and decode the received bytes.
+
+## Prerequisites
+- Python 3.6 or later.
+- **Microsoft Azure Subscription:**  To use Azure services, including Azure Schema Registry, you'll need a subscription.
+If you do not have an existing Azure account, you may sign up for a free trial or use your MSDN subscriber benefits when you [create an account](https://account.windowsazure.com/Home/Index).
+
+## Setup
+
+1. Install the Azure Schema Registry Avro Encoder client library and Azure Identity client library for Python with [pip](https://pypi.org/project/pip/):
+
+```bash
+pip install azure-schemaregistry-avroencoder azure-identity
+```
+
+Additionally, if using with `azure.eventhub.EventData`, install `azure-eventhub>=5.9.0b1`:
+
+```bash
+pip install azure-eventhub>=5.9.0b1
```

Comment:
nit: I would pin to version ==5.9.0b1 since EH is a preview, maybe there is new stable feature coming in 5.10 before the GAing protocol.
```suggestion
pip install azure-eventhub==5.9.0b1
```